SimGeneral/MixingModule/python/aliases_cfi.py

```diff
@@ -75,3 +84,6 @@
         2 : dict(type = "PHGCSimAccumulator"),
     }
 )
+
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
+(~phase2_hfnose).toModify(simHFNoseUnsuppressedDigis, mix = None)
```

Comment:
I wonder (in more general than just this PR since the same is done for HGCal as well in above) whether it would be more clear to define an empty `EDAlias` by default and enable it with a `Modifier` rather than disabling the defined-by-default `EDAlias` with an inverted `Modifier`.

Reply:
As far as I'm concerned yes. My intention was more pose a question (to wider audience) which way we'd prefer to define these aliases.

Reply:
I see benefits to each approach:
* if you're looking at the overall workflow, it's better to see the new subdetectors with `None` by default
* if you're looking specifically at one of the subdetectors, it's nicer to see right away how the alias is defined
so I don't have a strong preference.